src/openms/include/OpenMS/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.h

```diff
@@ -60,8 +61,8 @@ namespace OpenMS
     /// Quantitative and associated data for a peptide
     struct PeptideData
     {
-      /// mapping: charge -> sample -> abundance
-      std::map<Int, SampleAbundances> abundances;
+      /// mapping: fraction -> charge -> sample -> abundance
+      std::map<Int, std::map<Int, SampleAbundances>> abundances;
```

Comment:
this may not wrap well in Python, you may have to do something like this
```suggestion
     struct FractionAbundances { std::map<Int, SampleAbundances> sample_abundances;};
      std::map<Int, FractionAbundances> abundances;
```

Reply:
so far it has not been wrapped: https://github.com/OpenMS/OpenMS/blob/develop/src/pyOpenMS/pxds/PeptideAndProteinQuant.pxd#L64
but I can add that.